src/config/ConfigDataValues.hpp

```diff
@@ -136,3 +137,18 @@ class CCssGapData : public ICustomConfigValueData {
         return std::format("{} {} {} {}", top, right, bottom, left);
     }
 };
+
+class CStringOrInt : public ICustomConfigValueData {
```

Comment:
I don't like this. Why not make this class a `CFontWeightConfigValueData` and have it take a `const char*` in the ctor and set its value on that?

Reply:
Class renamed as requested. The reason I named it like I did was because I'd hoped to reuse it for other config values that might benefit from having a human-readable value in the config file, but an integer representation in the code. Admittedly, this feels like a sloppy solution, and I still think there's _some_ value to allowing `CConfig::addConfigValue` to take an optional parse function. For now though, this is good enough.